public/index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Origin1</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Arapey:400italic' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Libre+Baskerville:400italic' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Palanquin|Rancho' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Palanquin' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Rajdhani' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="vendors/styles/normalize.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/layout.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/home.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/contact.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/logo.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/coffeepage.css" media="screen" charset="utf-8">
+
+
+  <script id="coffeePageTemplate" type="text/x-handlebars-template">
+    <section id="coffeePageContainer">
+      <section class="coffeePages" id="featuredCoffee">
+
+      <section class="sideBar featuredEls" id="regionInfo"><h1 id="countryName">{{contryName}}</h1><h3 id="regionBio">{{countryBio}}</h3></section>
+        <section id="roasterHead">
+          <h1 id="roasterH1">The Cup we like!</h1>
+          <img id ="downArrow" class="arrows" src="/media/images/downArrow.png" alt="arrow" />
+        </section>
+        <section class="sideBar featuredEls roaster" id="roasterInfo">{{featuredRoasterBio}}</section>
+        <section class="featuredEls roaster" id="goldenCoffee">
+          <img src="{{featuredCoffeePic}}" alt="not found" />
+        </section>
+    </section>
+
+    <section id="mapHead">
+      <h1 id="mapH1">The matter of Weather</h1>
+      <img id ="mapDownArrow" class="arrows" src="/media/images/mapDownArrow.png" alt="arrow" />
+    </section>
+    <section class="coffeePages" id="coffeeMap">
+      <section id="mapContainer">
+        <ul id="mapMonths">
```

Comment:
These elements didn't style well on the page. You could consider shortening the months to a standard 3-character usage; eg. Jan, Feb, Mar, ...which would make things easier for you to style and standardize.